engine/src/test/java/org/camunda/bpm/engine/test/api/multitenancy/query/MultiTenancyUserOperationLogQueryTest.java

```diff
@@ -0,0 +1,223 @@
+
+    // then
+    assertThat(singleResult).isEqualTo(null);
+  }
+
+  @Test
+  public void shouldReturnResultsWhenMultipleTenants() {
+    // given task with no tenant, with TENANT_ONE, and with TENANT_TWO
+    testRule.deploy(MODEL);
+    runtimeService.startProcessInstanceByKey(PROCESS_NAME);
+    identityService.setAuthenticatedUserId(USER_WITHOUT_TENANT);
+    String taskId = taskService.createTaskQuery().singleResult().getId();
+    taskService.complete(taskId);
+
+    testRule.deployForTenant(TENANT_ONE, MODEL);
+    identityService.setAuthentication(USER_ONE, null, Arrays.asList(TENANT_ONE));
+    runtimeService.createProcessInstanceByKey(PROCESS_NAME)
+    .processDefinitionTenantId(TENANT_ONE)
+    .execute();
+    taskId = taskService.createTaskQuery().tenantIdIn(TENANT_ONE).singleResult().getId();
+    taskService.complete(taskId);
+
+    testRule.deployForTenant(TENANT_TWO, MODEL);
+    identityService.setAuthentication(USER_TWO, null, Arrays.asList(TENANT_TWO));
+    runtimeService.createProcessInstanceByKey(PROCESS_NAME)
+    .processDefinitionTenantId(TENANT_TWO)
+    .execute();
+    taskId = taskService.createTaskQuery().tenantIdIn(TENANT_TWO).singleResult().getId();
+    taskService.complete(taskId);
+
+    // when query with USER_TWO
+    List<UserOperationLogEntry> list = historyService.createUserOperationLogQuery()
+        .entityType(EntityTypes.TASK)
+        .list();
+
+    // then
+    assertThat(list.size()).isEqualTo(2);
+    assertThat(list.get(0).getTenantId()).isIn(null, TENANT_TWO);
+    assertThat(list.get(1).getTenantId()).isIn(null, TENANT_TWO);
+  }
+
+  @Test
+  public void shouldReturnResultsWhenTwoTenant() {
+    // given logs with assigned tenant
+    // and user belonging to two tenants
+    testRule.deployForTenant(TENANT_ONE, MODEL);
+    identityService.setAuthentication(USER_ONE, null, Arrays.asList(TENANT_ONE, TENANT_TWO));
+
+    runtimeService.startProcessInstanceByKey(PROCESS_NAME);
+    String taskId = taskService.createTaskQuery().singleResult().getId();
+
+    taskService.complete(taskId);
+
+    // when
+    UserOperationLogEntry singleResult = historyService.createUserOperationLogQuery()
+        .entityType(EntityTypes.TASK)
+        .singleResult();
+
+    // then
+    assertThat(singleResult.getTenantId()).isEqualTo(TENANT_ONE);
+  }
+
+  @Test
+  public void shouldReturnResultsWithoutTenantId() {
+    // given task with no tenant, with TENANT_ONE, and with TENANT_TWO
+    testRule.deploy(MODEL);
+    runtimeService.startProcessInstanceByKey(PROCESS_NAME);
+    identityService.setAuthenticatedUserId(USER_WITHOUT_TENANT);
+    String taskId = taskService.createTaskQuery().singleResult().getId();
+    taskService.complete(taskId);
+
+    testRule.deployForTenant(TENANT_ONE, MODEL);
+    identityService.setAuthentication(USER_ONE, null, Arrays.asList(TENANT_ONE));
+    runtimeService.createProcessInstanceByKey(PROCESS_NAME)
+    .processDefinitionTenantId(TENANT_ONE)
+    .execute();
```

Comment:
Indentation seems to be off here.
```suggestion
        .processDefinitionTenantId(TENANT_ONE)
        .execute();
```